Easy/1. Shopping Cart Problem/solutions/shoppingcart.py

```diff
@@ -0,0 +1,26 @@
+products = [("Basshead earphones",1200),
+                ("Bluetooth mouse",600)
+]
+
+def show_products():
+    print("Press the corresponding number for the product")
```

Comment:
Open Source Session Testing Comment

Reply:
Okay will make the changes. Making tax rate to 6%